### PR TITLE
fix: sticky sidebar on checkout page no longer overlaps footer

### LINKED ISSUE #2586: [bug]: Sticky register form at Receipt screen is overwritten by footer on resolution 1440px
**Describe the bug**
[Desktop resolution: 1440x900] After place order with guest successfully. At the Receipt screen, scroll page to the bottom the we found the right register form is overwritten by footer.

**To reproduce**
1. Add product to cart
2. Go to checkout page
3. Fill all required fields and Place order
4. At Receipt screen, scroll to bottom 
5. See error in the right page

**Expected behavior**
Fill background for register form section or make it bellow the footer.

**Screenshots**
Screen Shot 2020-07-29 at 3 31 55 PM

**Additional context**

**Possible solutions**
Add any ideas about possible solutions to the problem here.

**Please complete the following device information:**
 - Device [e.g. iPhone6, PC, Mac, Pixel3]: Macbook
 - OS [e.g. iOS8.1, Windows 10]: MacOS 10.14.6
 - Browser [e.g. Chrome, Safari]: Chrome 
 - Browser Version [e.g. 22]:
 - Magento Version: 2.4
 - PWA Studio Version: 7.0.0
 - NPM version `npm -v`: 6.9.0
 - Node Version `node -v`: v10.16.0

**Please let us know what packages this bug is in regards to:**
- [ ] `venia-concept`
- [x] `venia-ui`
- [ ] `pwa-buildpack`
- [ ] `peregrine`
- [ ] `pwa-devdocs`
- [ ] `upward-js`
- [ ] `upward-spec`
- [ ] `create-pwa`

### PR DESCRIPTION
**Pull request by sirugh**
## Description

Create Account sidebar was overlapping with footer.

## Related Issue

* Closes [PWA-784](https://jira.corp.magento.com/browse/PWA-784).
* Closes #2586.

### Verification Steps

1. Open dev tools, switch to ipad view, rotate to landscape.
2. Check out and get to order conf page.
3. Scroll down, ensure that sidebar/create account does not overlap with footer

## Screenshots / Screen Captures (if appropriate)

[Image from Gyazo](https://gyazo.com/e1ffcec11fa9449fc9f5714140a205a9)

## Checklist

- [ ] I have added tests to cover my changes, if necessary.
* I have updated the documentation accordingly, if necessary.

### CODE REVIEW

**Comment by devops-pwa-codebuild** [comment]
**Performance Test Results**

The following fails have been reported by WebpageTest. These numbers indicates a possible performance issue with the PR which requires further manual testing to validate.

https://pr-2582.pwa-venia.com : LH Performance Expected 0.85 Actual 0.52, LH Best Practices Expected 1 Actual 0.92, WPT Cache Expected 90 Actual 83.333333333333
https://pr-2582.pwa-venia.com/venia-tops.html : LH Performance Expected 0.75 Actual 0.34, LH Best Practices Expected 1 Actual 0.92
https://pr-2582.pwa-venia.com/valeria-two-layer-tank.html : LH Performance Expected 0.8 Actual 0.41, LH Accessibility Expected 0.9 Actual 0.89, LH Best Practices Expected 1 Actual 0.92, WPT Cache Expected 65 Actual 50

**Comment by schensley** [comment]
@sirugh great work. It looks fixed. Thank you. UX Approved.

File: packages/venia-ui/lib/components/CheckoutPage/OrderConfirmationPage/createAccount.css [hunk 1]
```diff
@@ -2,6 +2,9 @@
     display: grid;
     grid-template-columns: 1fr;
     gap: 1rem;
+
+    position: sticky;
```
Comment (tjwiebell) [inline packages/venia-ui/lib/components/CheckoutPage/OrderConfirmationPage/createAccount.css:6]: I somewhat expected this to behave oddly on mobile, but it appears the stickiness just never kicks in. I did observe some odd behavior when toggling `position: sticky` on and off; consider explicitly setting back to `position: relative` and `top: auto` in mobile rule.

Comment (sirugh) [reply packages/venia-ui/lib/components/CheckoutPage/OrderConfirmationPage/createAccount.css:6]: I wasn't seeing anything happen when I toggled the sticky position but I think it's fine to reset to default on mobile where the styles no longer apply.

**Pull request merged**